### DISCUSSION TITLE
How to determine correct reconnect time for solving recaptchas in UC mode?

### DISCUSSION THREAD

**Discussion by krzysztofkub** [seleniumbase/SeleniumBase#3314] [Q&A] [answered]
How, I was wondering. How are you finding correct value of reconnect param? In tutorials @mdmintz says to find correct value for site, but in examples there is for example reconnect time 6.111. Are you incrementing by 0.001 and check? For me sometimes I get extra pictures to solve even if I use 20 (sometimes)

**Comment by mdmintz** [comment, marked as answer]
Regular UC Mode is being replaced by [CDP Mode](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/cdp_mode/ReadMe.md).
Before, you had to guess an optimal `reconnect_time`, which depends on internet connection speed.

**Reply by krzysztofkub** [reply]
but the CDP Mode doesn't have catpcha handler

**Reply by mdmintz** [reply]
CDP Mode is a subset of UC Mode, so it does have access to UC Mode methods. Eg:

```python
from seleniumbase import SB

with SB(uc=True, test=True, locale_code="en", incognito=True) as sb:
    url = "https://wsform.com/demo/"
    sb.activate_cdp_mode(url)
    sb.scroll_into_view("div.grid")
    sb.uc_gui_click_captcha()
    sb.sleep(1)
```
